go.mod

```diff
@@ -1,95 +1,97 @@
 module github.com/crunchydata/postgres-operator
 
-go 1.19
+go 1.22.0
+
+toolchain go1.22.4
 
 require (
-	github.com/evanphx/json-patch/v5 v5.6.0
-	github.com/go-logr/logr v1.3.0
+	github.com/evanphx/json-patch/v5 v5.9.0
+	github.com/go-logr/logr v1.4.2
 	github.com/golang-jwt/jwt/v5 v5.2.1
-	github.com/google/go-cmp v0.5.9
-	github.com/google/uuid v1.3.1
-	github.com/onsi/ginkgo/v2 v2.0.0
-	github.com/onsi/gomega v1.18.1
+	github.com/google/go-cmp v0.6.0
+	github.com/google/uuid v1.6.0
+	github.com/onsi/ginkgo/v2 v2.17.2
+	github.com/onsi/gomega v1.33.1
 	github.com/pganalyze/pg_query_go/v5 v5.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.3
 	github.com/xdg-go/stringprep v1.0.2
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.45.0
-	go.opentelemetry.io/otel v1.19.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.2.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.2.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.49.0
+	go.opentelemetry.io/otel v1.27.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace v1.27.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.27.0
 	go.opentelemetry.io/otel/exporters/stdout/stdouttrace v1.2.0
-	go.opentelemetry.io/otel/sdk v1.2.0
-	go.opentelemetry.io/otel/trace v1.19.0
-	golang.org/x/crypto v0.22.0
+	go.opentelemetry.io/otel/sdk v1.27.0
+	go.opentelemetry.io/otel/trace v1.27.0
+	golang.org/x/crypto v0.24.0
 	gotest.tools/v3 v3.1.0
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
-	k8s.io/component-base v0.24.2
-	sigs.k8s.io/controller-runtime v0.12.3
-	sigs.k8s.io/yaml v1.3.0
+	k8s.io/api v0.30.2
```

Comment:
❓ Do you have plans to bump controller-gen as well? I vaguely remember that we wanted or needed the K8s version to match across these tools. It looks like [v0.15.0](https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.15.0) is using k8s 1.30.

https://github.com/CrunchyData/postgres-operator/blob/94898c516c6e460f0eb389646221ab62e967a1ed/Makefile#L320

Reply:
Created a story to investigate a potential `controller-tools` bump